### PR TITLE
Add selectSort in format of .java

### PR DESCRIPTION
**Pull request by 8Malekian**
The line with java package have been removed to avoid any conflict during the merge

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Oui on va voir ça!

File: SelectSort.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+public class SelectSort {
```
Comment (cegerard) [inline SelectSort.java]: This class must implements the ISortingAlgorithm interface.

File: SelectSort.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	                    index = j;
+	      
+	            int smallerNumber = arr[index];  
+	            arr[index] = arr[i];
+	            arr[i] = smallerNumber;
+	        }
+	        return arr;
+	    }
+	     
+	    public static void main(String a[]){
```
Comment (cegerard) [inline SelectSort.java]: You can get rid of this useless method.
We use the main method from the App class as a starting point.

**Review by cegerard — CHANGES_REQUESTED** [review]
You must add the algorithm package at the top of this file.
`package sortBench.algorithm`

You must move this file into the right directory.
src/main/java/sortBench/algorithms/select

File: SelectSort.java (hunk 1 above)
Comment (cegerard) [reply SelectSort.java]: You can delete this file

File: src/main/java/sortBench/algorithms/select/SelectSort.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm
+
+public class SelectSort {
```
Comment (cegerard) [inline src/main/java/sortBench/algorithms/select/SelectSort.java]: replace 
`public class SelectSort {`
by
`public class SelectSort implements ISortingAlgorithm {`

Comment (8Malekian) [reply src/main/java/sortBench/algorithms/select/SelectSort.java]: Well the push steps works for now....

File: src/main/java/sortBench/algorithms/select/SelectSort.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm
+
+public class SelectSort implements ISortingAlgorithm {
+	
+ 	 
+	    private void doSelectionSort(int[] arr){
```
Comment (cegerard) [inline src/main/java/sortBench/algorithms/select/SelectSort.java]: To match interface implementation just rename this method in `sort`

Comment (cegerard) [reply src/main/java/sortBench/algorithms/select/SelectSort.java]: After that test will be ok.

File: src/main/java/sortBench/algorithms/select/SelectSort.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm
```
Comment (cegerard) [inline src/main/java/sortBench/algorithms/select/SelectSort.java]: Here you forgot the `;` at the end of the line

**Review by cegerard — CHANGES_REQUESTED** [review]

**Review by cegerard — APPROVED** [review]

File: src/main/java/sortBench/algorithms/select/SelectSort.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm;
+
+public class SelectSort implements ISortingAlgorithm {
+	
+ 	 
+	    private void sort(int[] arr){
```
Comment (cegerard) [inline src/main/java/sortBench/algorithms/select/SelectSort.java]: Oups this method must be public not private to make this class ready to test.

**Review by cegerard — CHANGES_REQUESTED** [review]

File: src/main/java/sortBench/algorithms/select/SelectSort.java (hunk 5 above)
Comment (8Malekian) [reply src/main/java/sortBench/algorithms/select/SelectSort.java]: working in notpad only is bad....working in eclips is better 

until I won't be a noob.

**Review by cegerard — APPROVED** [review]

**Comment by cegerard** [comment]
OK Good you can merge now.

**Pull request merged**